db/migrate/20220920110257_add_post_ref_to_likes.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPostRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :post, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Please follow the ERD provided for this tutorial, right now you are not affecting the Author as a reference for the user.